Mathlib/RingTheory/Algebraic.lean

```diff
@@ -483,17 +486,20 @@ theorem Transcendental.of_tower_top {x : A} (h : Transcendental L x) :
     Transcendental K x := fun H ↦ h (H.tower_top L)
 
 /-- If A is an algebraic algebra over K, then A is algebraic over L when L is an extension of K -/
+@[stacks 09GF "part two"]
 theorem Algebra.IsAlgebraic.tower_top [Algebra.IsAlgebraic K A] : Algebra.IsAlgebraic L A :=
   Algebra.IsAlgebraic.tower_top_of_injective (algebraMap K L).injective
 
 variable (K)
 
+@[stacks 09GG]
 theorem IsAlgebraic.of_finite (e : A) [FiniteDimensional K A] : IsAlgebraic K e :=
   (IsIntegral.of_finite K e).isAlgebraic
 
 variable (A)
 
 /-- A field extension is algebraic if it is finite. -/
+@[stacks 09GG]
```

Comment:
I don't think this tag is right.

Reply:
I believe this theorem aligns with the first part `A finite extension is algebraic` of the 09GG statement.

A finite extension is algebraic. In fact, an extension E/k is algebraic if and only if every subextension k(α)/k generated by some α∈E is finite.

I'll add a comment saying it is the first part.